src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -1,14 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <IncludeInPackage>Microsoft.ML</IncludeInPackage>
+    <Configurations>Debug;Release;Debug-Intrinsics;Release-Intriniscs</Configurations>
```

Comment:
Who sets this configuration?

Reply:
Side note: "Release-Intriniscs" could probably be "Release-Intrinsics".

Reply:
I see, and you aren't going to turn it on in CI or official build yet?

Reply:
For sure, "no" for official build.  I don't want to affect the official product until we are ready to affect it (3.0 doesn't even have a preview out yet, so getting it into customers hands isn't a critical concern for me).

We will eventually turn CI on for it, especially as we move closer to wanting to put it in the official product.